applications/surgical_scene_recon/training/scene/endo_loader.py

```
@@ -95,15 +95,58 @@ def load_meta(self):
+        # get paths of images, depths, masks first to determine frame count
+        def get_file_paths(filetype):
+            """Get sorted list of PNG files for a given data type."""
+            return sorted(glob.glob(os.path.join(self.root_dir, filetype, "*.png")))
+
+        self.image_paths = get_file_paths("images")
+        if self.mode == "binocular":
+            self.depth_paths = get_file_paths("depth")
+        elif self.mode == "monocular":
+            self.depth_paths = get_file_paths("monodepth")
+        else:
+            raise ValueError(f"{self.mode} has not been implemented.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)